### PR TITLE
Fix 252: adding support for the helm repository type

### PR DESCRIPTION
**Pull request by glockhart**

### CODE REVIEW

**Comment by glockhart** [comment]
@eyalbe4 are there comments / blockers on this PR that I need to address?
You should have a signed CLA already, so please let me know if you need anything else

**Comment by eyalbe4** [comment]
@glockhart,
Thanks for this addition.
I suggest we add tests for the new package type.
You can look at the existing tests as a reference. For example, see the *NpmPackageTypeRepositoryTests*.

File: services/src/main/java/org/jfrog/artifactory/client/model/repository/settings/impl/HelmRepositorySettingsImpl.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    public HelmRepositorySettingsImpl() {
+        super(defaultLayout);
+    }
+
+    public PackageType getPackageType() {
+        return PackageTypeImpl.helm;
+    }
+
+    
+
```
Comment (eyalbe4) [inline services/src/main/java/org/jfrog/artifactory/client/model/repository/settings/impl/HelmRepositorySettingsImpl.java]: Let's remove the redundant new-lines in this class.

Comment (glockhart) [reply services/src/main/java/org/jfrog/artifactory/client/model/repository/settings/impl/HelmRepositorySettingsImpl.java]: Done

**Comment by glockhart** [comment]
@eyalbe4 review comments have been addressed

**Pull request merged**